packages/web/pages/assets/index.tsx

```diff
@@ -157,18 +169,26 @@ const Assets: NextPage = observer(() => {
       <AssetsTable
         nativeBalances={nativeBalances}
         ibcBalances={ibcBalances}
-        onDepositIntent={() => transferConfig.startTransfer("deposit")}
-        onWithdrawIntent={() => transferConfig.startTransfer("withdraw")}
-        onDeposit={(chainId, coinDenom, externalDepositUrl) => {
+        onDepositIntent={() => {
+          transferConfig.startTransfer("deposit");
+          setTransferQueryParams("deposit");
+        }}
+        onWithdrawIntent={() => {
+          transferConfig.startTransfer("withdraw");
+          setTransferQueryParams("withdraw");
+        }}
+        onDeposit={(coinDenom, externalDepositUrl) => {
```

Comment:
I think the cause of the bug was removing the first param here. I'd add it back with an `_` rather than go into the assets table and edit the code. Chain ID could be useful in the far future if a chain is insistent on having the same token denom as a token on another chain.

Applies to `onWithdraw` below as well.